src/test/py/bazel/bazel_windows_symlinks_test.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import unittest
+from src.test.py.bazel import test_base
+
+
+class BazelWindowsTest(test_base.TestBase):
+
+    def createProjectFiles(self):
+        self.CreateWorkspaceWithDefaultRepos('WORKSPACE')
+        self.ScratchFile(
+            'foo/BUILD', ['genrule(name="x", srcs=[":sample"], outs=["link"], cmd="python -c \\"import os; os.symlink(os.path.join(os.getcwd(), \'$<\'),\'$@\')\\"",)'])
```

Comment:
Don't assume `python` is on the PATH, it isn't always.
Either create a py_binary with this script and declare it in genrule.tools and run the py_binary from the genrule.cmd, or run `in=$<; out=$@; cmd.exe /C mklink ${in//\//\\} ${out//\//\\}`.